indico/modules/events/templates/display/conference/base.html

```diff
@@ -31,7 +35,7 @@ <h1>
                                        <img src="{{ event.logo_url }}" alt="{{ event.title }}" border="0" class="confLogo">
                                     </div>
                                 {% endif %}
-                                <span itemprop="title">{{ event.title }}</span>
+                                <span itemprop="title" itemprop="name">{{ event.title }}</span>
```

Comment:
is having two `itemprop` attributes valid?

Reply:
Probably not. Only name should be left( that's what both cement and schema.org Event specify.)

Reply:
Done